src/libraries/System.Runtime.InteropServices.RuntimeInformation/tests/System.Runtime.InteropServices.RuntimeInformation.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <IncludeRemoteExecutor>true</IncludeRemoteExecutor>
-    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
```

Comment:
Why? I see multiple tfms in the src project: https://github.com/dotnet/runtime/blob/613fb3b808dcd2336e9e50d4d6191b8c4923c7c4/src/libraries/System.Runtime.InteropServices.RuntimeInformation/src/System.Runtime.InteropServices.RuntimeInformation.csproj#L6

Reply:
yes this needs to be reverted & browser rid needs to be added

Reply:
Is this because we're moving to P2Ps? Because with the current way everything is built, it shouldn't be necessary to match test projects with src projects.

Reply:
> @ViktorHofer shouldnt the test build fail as src project doesnt have any ridless tfm ?

currently this is not enforced as the configuration system gets the assets from the current OS binplaced into the ridless runtime and testhost directory.

> Is this because we're moving to P2Ps? Because with the current way everything is built, it shouldn't be necessary to match test projects with src projects.

The recommendation was always to keep the test project configurations in sync with the source ones but it wasn't enforced. Now with switching to P2Ps this will be enforced as the configuration will fail if an OS agnostic configuration tries to reference a non-agnostic one.